homeassistant/components/homewizard/manifest.json

```diff
@@ -12,6 +12,6 @@
   "iot_class": "local_polling",
   "loggers": ["homewizard_energy"],
   "quality_scale": "platinum",
-  "requirements": ["python-homewizard-energy==v7.0.1"],
+  "requirements": ["python-homewizard-energy==v8.0.0b3"],
```

Comment:
Can we actually release a full version rather than a beta? I saw an issue last week where `uv` apparently blocks installing pre-release dependencies if the main package (HA) isn't a pre-release as well